cmd/kubeadm/app/constants/constants.go

```diff
@@ -309,7 +309,7 @@ const (
 	KubeDNSVersion = "1.14.13"
 
 	// CoreDNSVersion is the version of CoreDNS to be deployed if it is used
-	CoreDNSVersion = "1.2.4"
+	CoreDNSVersion = "1.2.6"
```

Comment:
The version in the [upgrade tests](https://github.com/kubernetes/kubernetes/blob/master/cmd/kubeadm/app/phases/upgrade/compute_test.go) also need to be changed.